docs/basics/multi-node.md

```diff
@@ -145,9 +145,7 @@ local reference to a remote graph:
 ```groovy
 gremlin> cluster = Cluster.open('conf/remote-objects.yaml')
 ==>localhost/127.0.0.1:8182
-gremlin> graph = EmptyGraph.instance()
-==>emptygraph[empty]
-gremlin> g = graph.traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
+gremlin> g = AnonymousTraversalSource.traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
```

Comment:
Still with `AnonymousTraversalSource.traversal()` here

Reply:
In this case i wasn’t sure how write it correctly.

Reply:
What do you mean? Did you try it out in the console? The console already has the static imports. So it just works:

```groovy
gremlin> cluster = Cluster.open('conf/remote-objects.yaml')
==>localhost/127.0.0.1:8182
gremlin> g = traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
==>graphtraversalsource[emptygraph[empty], standard]
```

Reply:
Do you like to review it again?